### DISCUSSION TITLE
How do I add tooltip to component with nested component?

### DISCUSSION THREAD

**Discussion by starxyn** [mantinedev/mantine#7274] [Q&A]
I have components like this:

```
<Component1>
   <Button />
</Component>
```

and Component1 has some logic to recognize the child component i.e. <Button>

I want to be able to set tooltip on the button but I couldn't seem to place it anywhere.

Wrapping <Tooltip> the Component1 or Button doesn't work.

**Comment by rtivital** [comment]
Wrap button with a tooltip
